sim/scenarios/drop-rate/drop-rate-error-model.h

```diff
@@ -1,23 +1,26 @@
 #ifndef DROPRATE_ERROR_MODEL_H
 #define DROPRATE_ERROR_MODEL_H
 
-#include <set>
-#include <random>
 #include "ns3/error-model.h"
 
 using namespace ns3;
 
-// The DropRateErrorModel drops random packets, at a user-specified drop rate. 
+// The DropRateErrorModel drops random packets, at a user-specified drop rate.
+// But no more than a user-specified number of packets in a row.
 class DropRateErrorModel : public ErrorModel {
  public:
     static TypeId GetTypeId(void);
     DropRateErrorModel();
     void SetDropRate(int perc);
-    
- private:
+    void SetMaxDropBurst(int burst);
+
+  private:
     int rate;
-    std::mt19937 *rng;
-    std::uniform_int_distribution<> distr;
```

Comment:
Any reason for switching out the random number generator? Not opposed to it, just curious.

Reply:
No real reason, was just wondering why we used such a complicated approach before.